src/Controllers/Admin/NewsController.php

```diff
@@ -44,6 +45,7 @@ public function edit(Request $request): Response
 
     protected function showEdit(?News $news, bool $sendNotification = true, bool $isMeetingDefault = false): Response
     {
+        $notifications_count = Settings::whereEmailNews(true)->count();
```

Comment:
As a variable name you should use `snakeCase`

Reply:
done